instruqt-tracks/sentinel-for-terraform-v4/01-restrict-vault-auth-methods/assignment.md

```diff
@@ -0,0 +1,176 @@
+---
+slug: restrict-vault-auth-methods
+id: ythxlywtqm9b
```

Comment:
Let's clear the `id:` fields out of `assignment.md` while we're moving to v3 of the track config